content/_help/get-started/the-deadline-to-verify-my-identity-in-person-passed._en.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: help
+title: The deadline to verify my identity in person passed
+category: get-started
+permalink: /help/get-started/the-deadline-to-verify-my-identity-in-person-passed/
+order: 10
```

Comment:
Can we decide what order we want these pages to appear in the navigation? All three of the new page content pull requests (#931, #932, and #933) use the same order `10`.

The main content in #946 is using order `7`, so if we want that to be first, we should assign `8`, `9`, and `10` to the three new content pages.